packages/babel-jest/src/index.js

```diff
@@ -52,7 +52,7 @@ const createTransformer = (options: any): Transformer => {
       configString: string,
       {config, instrument, rootDir}: {config: ProjectConfig} & CacheKeyOptions,
     ): string {
-      const babelOptions = loadBabelConfig(config.cwd, filename);
+      const babelOptions = loadBabelConfig(config.rootDir, filename);
```

Comment:
IMO making `rootDir` a `cwd` conflates the two concepts a bit. Can we specify `root` or `babelrcRoots` or something to Babel to achieve the same thing?

Reply:
`babelrcRoots` is wrong. `root` is probably correct (and what I suggested in https://github.com/facebook/jest/issues/7438#issuecomment-451309256 - `root: config.rootDir`)

Reply:
Feedback being

>I don't think I'd set `root` because it's really more dependent on user project layout, so it's something I'd probably expect the user to set. If that's getting easier, that's excellent too though.

But if we _do_ require to user to set `rootDir`, then I think we're good to go

Reply:
The question is whether to use babel's  [`cwd`](https://babeljs.io/docs/en/options#cwd) or [`root`](https://babeljs.io/docs/en/options#root).

I think `cwd` is the correct option to use to achieve exactly the same behavior as running individually.

Other than being used as the default for `root`, it is also used as the base for [relative filenames](https://babeljs.io/docs/en/options#filenamerelative).  I'm not sure if there's anything else (resolving plugins?), but I think we're likely to run into subtle issues if we use `root` instead of `cwd`.

The feedback quoted in comment above was from @loganfsmyth, and it sounds like he is suggesting to use `cwd`.

Reply:
Realistically, what I'd probably do is have the initial call to `createTransformer()` call `process.cwd()` in https://github.com/facebook/jest/blob/b16789230fd45056a7f2fa199bae06c7a1780deb/packages/babel-jest/src/index.js#L30 as 
```
  cwd: process.cwd(),
   ...options
```
That way user's doing things like:

```
module.exports = require("babel-jest").createTransformer({
  plugins: ['@babel/preset-react'],
});
```
end up with the preset consistently resolved relative to the working directory at the time the transformer was created. The core goal of `cwd` is to define the root directory to use when resolve relative paths passed into Babel's options. I'd then expect Jest to pass `filename` as an absolute path, so that the `cwd` doesn't matter.

I can see the argument for setting `root` based on `rootDir` and if that's the way Jest wants to go I don't think it's unreasonable, as long as it is documented as a breaking change for users upgrading to new versions. Ideally users should still be able to do

```
module.exports = require("babel-jest").createTransformer({
  root: __dirname,
});
```
to override Jest's default `root` value and set an explicit one in Babel's options.